### PR TITLE
Entertainer Drip (Mime/Clown)

### PR DESCRIPTION
**Pull request by Christasmurf**
## What Does This PR Do
Ports the mime outfits from tg https://github.com/tgstation/tgstation/pull/64433
Minor resprite for the clown (better shading), addition of a clown skirt. Rebrands the sexy clown outfit.
Resprites the clown backpack/duffel, adds a clown satchel

Species sprites for the sexy outfits made by Synth

## Why It's Good For The Game
Better looking sprites

## Images of changes
Before:
image

After:
image
(The mime uniform/skirt can be rolled down)

Bags:
image

Duffelbag item change for better shading
image

## Testing
Spawn in and check lockers, wear outfits with magic mirror for species sprites

## Changelog
:cl: Christasmurf, Synth
add: Added clown skirt, clown satchel, ported mime skirt
imageadd: Ported mime jumpsuit/sexy from tg, resprited clown sexy, minor touch-ups to clown uniform + bags resprited
/:cl:

### CODE REVIEW

**Comment by PhantornRU** [comment]
The previous outfits are good. It's better not to delete them, but to create new objects with new/old sprites.

**Comment by Christasmurf** [comment]
> The previous outfits are good. It's better not to delete them, but to create new objects with new/old sprites.

The previous outfits are old, and crusty, and people have expressed a desire to see them change because of how bad they are. If you wish to keep them for personal use, download them from the repo now because I have no intention of keeping them with this PR.

**Comment by Probablyjay19** [comment]
> > The previous outfits are good. It's better not to delete them, but to create new objects with new/old sprites.
> 
> 
> 
> The previous outfits are old, and crusty, and people have expressed a desire to see them change because of how bad they are. If you wish to keep them for personal use, download them from the repo now because I have no intention of keeping them with this PR.

Sadge, although I will miss the female mime performer outfit, that one is actually baller.

File: code/modules/clothing/under/jobs/civilian.dm [hunk 1]
```diff
@@ -90,9 +90,16 @@
...
+/obj/item/clothing/under/rank/civilian/clown/skirt
+	name = "clown skirt"
+	desc = "<i>'HONK!'</i>"
+	icon_state = "clown_skirt"
+	item_state = "clown_skirt"
+	item_color = "clown_skirt"
+
 /obj/item/clothing/under/rank/civilian/clown/sexy
 	name = "sexy-clown suit"
-	desc = "It makes you look HONKable!"
```
Comment (Henri215) [inline code/modules/clothing/under/jobs/civilian.dm:95]: 1984 again

**Review by Henri215 — APPROVED** [review]
LHTM (Looks Honkable To Me)

File: code/modules/clothing/under/jobs/civilian.dm (hunk 1 above)
Comment (Christasmurf) [reply code/modules/clothing/under/jobs/civilian.dm:95]: do not honk the clown

**Review by Burzah — APPROVED** [review]
Honk

**Comment by spacegeese** [comment]
How does this look with the completed clown outfit?

**Comment by Christasmurf** [comment]
> How does this look with the completed clown outfit?

image

image

image

**Comment by spacegeese** [comment]
HOLY SHIT
image
IT'S THE FUCKING
FUCKING LAWYER CLOWN

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**